core/storage/user/gae_models_test.py

```diff
@@ -45,6 +45,28 @@ def test_get_by_role(self):
         self.assertEqual(user[0].role, feconf.ROLE_ID_ADMIN)
 
 
+class StoryProgressModelTests(test_utils.GenericTestBase):
+
+    def test_get_multi(self):
+        model = user_models.StoryProgressModel.create(
+            'user_id', 'story_id_1')
+        model.user_id = 'user_id'
+        model.story_id = 'story_id_1'
```

Comment:
Why do you need to manually set fields on the model instance? Shouldn't create() handle that?

Reply:
Removed the lines